### DISCUSSION TITLE
Mindset: The New Psychology of Success

### DISCUSSION THREAD

**Discussion by agiletkiewicz** [Virtual-Coffee/virtualcoffee.io#1173] [VC Book Club]
[**Mindset: The New Psychology of Success by Carol S. Dweck**](https://www.goodreads.com/en/book/show/40745)

Every Friday, I'll post discussion questions in this thread and provide a link in Slack. Feel free to join in and participate in ways that are meaningful to you. You don't have to start at the same time as everyone else. Consider this an async book club that you can jump into at any point.

### SCHEDULE:
May 17: Chapter 1: THE MINDSETS & Chapter 2: INSIDE THE MINDSETS
May 24: Chapter 3: THE TRUTH ABOUT ABILITY AND ACCOMPLISHMENT & Chapter 4: SPORTS: THE MINDSET OF A CHAMPION
May 31: Chapter 5: BUSINESS: MINDSET AND LEADERSHIP
June 7:  Chapter 6: RELATIONSHIPS: MINDSETS IN LOVE (OR NOT) & Chapter 7: PARENTS, TEACHERS AND COACHES: WHERE DO MINDSETS COME FROM?
June 14: Chapter 8:  CHANGING MINDSETS

**Comment by agiletkiewicz** [comment]
Chapter 1 discussion topics:

* Reflect on the traits of fixed mindset and the growth mindset as outlined in Chapter 1. Which one do you see yourself leaning toward and do you think that your "personality mindset" and "intelligence mindset" differ? 
* How has each mindset (growth vs fixed) affected your approach to a particular situation or goal?
* How do you think these mindsets influence our long-term outcomes and fulfillment?
* When Dweck is reflecting on children’s responses to challenges, she says, "Not only weren’t they discouraged by failure, they didn’t even think they were failing. They thought they were learning." How does this compare to your reactions to failing? 
* What is a quote or section of this chapter that has changed/challenged your thinking? 


Chapter 2 discussion topics:

* Carol Dweck poses the question: "If you had to choose, which would it be? Loads of success or lots of challenge"? What is your reaction to this in the context of your own experience and the section on Is Success About Learning - Or Proving That You Are Smart?
* Reflect on the concept of "failure" as presented in Chapter 2 and your own relationship with failure. How do individuals with fixed mindsets versus growth mindsets perceive failure, and how does it influence their future actions and decisions?
* Did any answers in the Q&A section surprise you or resonate with you? Why?
* Consider the strategies and practices suggested in Chapter 2 for cultivating a growth mindset. Which strategies resonate with you the most, and how do you plan to incorporate them into your life or work?
* Chapter 2 is full of vignettes, many of which describe the mindsets and lives of famous people. Was there one story that really resonated with you? Why do you think that is?

**Reply by BekahHW** [reply]
I haven't found the audiobook so I may have to order a physical book, but I think this is a great resource and can't wait to get started.

**Reply by jonathanyeong** [reply]
I really liked the Q&A section in Chapter 2. As I was reading the first chapter, I kept telling myself that I think that you can have more than just a fixed OR a growth mindset. And she covers that answer in Chapter 2! I always felt like I had a growth mindset when it came to programming, but when it comes to creative endeavours, I feel like I have a fixed mindset. Another Q&A that resonated with me was when she talked about fixed mindset people working hard for validation versus for growth. I was like "oohhh that's me with creative stuff". I want to get the external validation of a good photo/piece of art/etc, and when it turns out poorly, I just give up. Which is expected since I've invested basically zero time into getting better 😅!

**Reply by savvycolleen** [reply]
@jonathanyeong, I also found it interesting to think about the different parts of your life that you may experience growth vs fixed mindset! I noticed that I was _majorly_ doing this in my personal life with exercise!! When I reflected on how I have previously been motivated to make and keep a routine, lately, I will find an excuse for anything and everything to assure myself why it's okay that I'm not doing it. It was a great revelation for me! Thanks for you sharing!

**Reply by dominicduffin1** [reply]
> Reflect on the traits of fixed mindset and the growth mindset as outlined in Chapter 1. Which one do you see yourself leaning toward and do you think that your "personality mindset" and "intelligence mindset" differ?

The more I learn about growth mindset the more I lean towards it, but I've not always been that way, its been a conscious choice to work on myself to achieve that.

> How do you think these mindsets influence our long-term outcomes and fulfillment?

I definitely agree with the book's premise that the growth mindset leads to better long-term outcomes and a more fulfilling life.

> When Dweck is reflecting on children’s responses to challenges, she says, "Not only weren’t they discouraged by failure, they didn’t even think they were failing. They thought they were learning." How does this compare to your reactions to failing?

I've long had a fear of failing in front of others while being happy to try things and fail and learn on my own. Its definitely a work in progress trying to become better at failing in public.

> "If you had to choose, which would it be? Loads of success or lots of challenge"?

I'd actually like a mixture of the two.

**Reply by agiletkiewicz** [reply]
@jonathanyeong what you said about seeking validation vs growth in certain areas resonates with me as well! I learned through getting good grades to chase that validation and get fulfillment from it. On the flip-side, @dominicduffin1 it's also been a conscious choice for me to work toward achieving a growth mindset and I have seen the benefits of working toward growth vs validation

**Comment by Lor1138** [comment]
My mindset has crippled me for as long as I can remember. When I started to learn about growth mindset that changed.

I found the section about failure to be particularly salient to this change. For so long failure or making mistakes was a death knell for anything I aimed for. Now when I struggle with something I take it as a sign that I am learning.

**Reply by savvycolleen** [reply]
@Lor1138 Yay, I love this!!

**Comment by savvycolleen** [comment]
We've got a lot of discussion questions available for this week's chapters, but feel free to pick and choose which ones you want to respond to! 

Chapter 3:

- Reflect on the idea of "the danger of praising intelligence." How does praising intelligence or talent impact individuals with fixed mindsets versus growth mindsets? Share examples from your own experiences or observations.
- Think about a specific skill or aspect of software development where you've noticed improvement over time. How did your mindset towards learning and development contribute to your progress in mastering this skill? Were there any setbacks or challenges along the way, and if so, how did you approach them?
- Explore the implications of the research findings regarding mindset and academic performance. How do mindsets influence students' attitudes towards learning, resilience in the face of challenges, and long-term academic success?
- Consider the practical strategies suggested in Chapter 3 for fostering a growth mindset in yourself or others. Which strategies do you find most effective, and how do you plan to implement them in your own life or work?
- Have you noticed a change in your thinking since starting this book? What language, stories, or new learning do you find yourself thinking about the most?
- Have you ever fallen prey to the "low effort syndrome"? How does the learning in this chapter shed light on why you may have done so and what you can do to avoid this pitfall?


Chapter 4:

- In Chapter 4, Dweck explores the mindset of athletes and its impact on performance. How do athletes with fixed mindsets versus growth mindsets approach challenges, setbacks, and competition? Share examples of athletes who embody each mindset.
- Reflect on a project or deadline where you experienced pressure similar to that felt by athletes in competition. How did your mindset—fixed or growth—affect your ability to manage stress, stay focused, and deliver results under pressure?
- Consider the parallels between the mindset required for success in sports and other areas of life, such as education, career, and personal development. What lessons can we learn from the mindset of champion athletes, and how can we apply them to our own pursuits and goals?
- Even after reading four chapters with research and examples, changing preconceived ideas can be hard. Is there a topic you are having trouble fully believing/embracing?
- In what ways have you seen/experienced "natural" ability be valued over individuals who work hard to gain their knowledge, talent, etc.?

**Reply by agiletkiewicz** [reply]
> Have you noticed a change in your thinking since starting this book?

One thing that I have been noticing recently is that I don't mind being corrected or asking questions in front of others. It's been a big unlock for me in terms of my professional growth because without this barrier I am introducing myself to new concepts more rapidly.

> Reflect on the idea of "the danger of praising intelligence." How does praising intelligence or talent impact individuals with fixed mindsets versus growth mindsets? Share examples from your own experiences or observations.

This is the biggest topic that hits home for me. I think that telling someone that they are smart comes with the unspoken rule that you can lose that status by not living up to it. Reflecting on my experiences, the more productive way has been praising performance. _"You got an A, you are so smart"_ implies having to continue to prove intelligence, but _"Great job putting in the work and getting an A"_ invites reflection on the actions and strategies that led to the result.

**Reply by savvycolleen** [reply]
Something that I am having trouble fully embracing is like.... where's the balance? Sometimes I feel like if I am working really hard in one (or more!) areas of my life, at least one facet of my life deserves some chill. Ha! For example, when I became a parent (at the beginning of the covid shutdown, mind you), I really felt like I deserved a break, especially when I started back to work. My mental health was a mess from being a new parent, working, and having a newborn in a pandemic, and I gave myself a lot of grace. Which I don't think is wrong! _But _ would Dweck say I was making excuses? Do I need to have the top most success in _every_ facet of my life at all times? Are there examples of people who embody a growth mindset who live more... normal (as in not famous) lives? Hopefully this makes sense. Thank you for listening to my wild brain dump.

**Reply by agiletkiewicz** [reply]
@savvycolleen I completely feel you - in fact, I originally rejected the very premise of the book based on the subtitle about "reaching your full potential". I thought - _what if I don't want to reach my full potential_?? The way that I have come to think about this for myself is that the shift here is that you _can_ be any kind of person that you like and achieve greatness in any area you like - but it does not mean you _should_ or _have to_. And that you can define what success and greatness looks like for you in each area. For example, I enjoy knitting and I am not striving to be the best knitter I can be. I am competent at the standard techniques and that's enough for me to get enjoyment and fulfillment out of it. I hope that offers something!

**Reply by savvycolleen** [reply]
@agiletkiewicz okay, yes! I love this! I really appreciate your thoughtful response. Thank you for helping me process that!!

**Comment by agiletkiewicz** [comment]
Chapter 5 discussion topics: 

- How have you observed the differences in leadership styles between those who exhibit a fixed mindset and those with a growth mindset in software development teams or companies you've been a part of? Can you share specific examples?
- Reflecting on your own experiences, how has feedback—both positive and negative—impacted your growth as a software developer? How do you believe a growth mindset approach to feedback could enhance team performance and individual growth?
- In your experience, how does a fixed mindset or growth mindset culture within a software development team influence innovation and problem-solving approaches? Can you recall a project where mindset played a significant role in its success or failure?
- Thinking about talent development within software development teams, how do you believe leaders can foster a culture that encourages continuous learning and embraces challenges? Can you share any strategies or initiatives that have worked well in your experience?

**Comment by agiletkiewicz** [comment]
Chapter 6 discussion topics:
* How do you think mindset influences the dynamics of romantic relationships, based on your own experiences or observations? Can you identify instances where a fixed mindset or a growth mindset played a role in the success or failure of a relationship?
* Consider the concept of "finding the right person" versus "developing the right relationship" as discussed in Chapter 6. How does this distinction relate to mindset, and how might it influence your approach to building and maintaining relationships in the future?
* Reflect on the strategies suggested in Chapter 6 for cultivating a growth mindset in relationships, such as viewing conflicts as opportunities for growth and learning. How do you plan to incorporate these strategies into your own approach to romantic relationships?

Chapter 7 discussion topics: 
* Reflecting on your upbringing and education, how do you think the mindsets of your parents, teachers, and coaches influenced your own mindset development as a software developer? Can you recall specific instances where their beliefs about intelligence and effort impacted your learning and growth?
* Dweck emphasizes the importance of praise and feedback in shaping children's mindsets. Looking back, how did the type of praise you received as a child—whether for academic achievements, creativity, or effort—affect your mindset and motivation to learn and succeed?
* Consider the role of failure and setbacks in learning and skill development, as discussed in Chapter 7. Can you recall a time when a failure or mistake taught you valuable lessons or spurred personal growth in your software development journey?

**Comment by meg-gutshall** [comment]
> Hi all! Our esteemed book club leaders are away at the moment, but asked me to post in their place. Let the discussion continue!

Chapter 8 discussion topics: **CHANGING MINDSETS**

- Think about a colleague or team member who has successfully transitioned from a fixed mindset to a growth mindset. What strategies or interventions do you believe were instrumental in facilitating this mindset shift, and how did it impact their performance and contribution to the team?
- Consider the concept of the "power of 'yet'" as discussed in Chapter 8. Can you think of instances where adopting a "yet" mindset—acknowledging that you haven't achieved something "yet," but you're capable of learning and growing—has fueled your motivation and resilience in pursuing challenging goals or mastering new skills?
- Reflect on the practical strategies suggested in Chapter 8 for fostering a growth mindset, such as embracing challenges, seeking feedback, and reframing setbacks as opportunities for learning. Which of these strategies resonate with you the most, and how do you plan to incorporate them into your daily practices?
